package.json

```diff
@@ -51,7 +51,7 @@
     "form-data": "^2.3.2",
     "go-ipfs-dep": "~0.4.17",
     "hat": "0.0.3",
-    "ipfs": "~0.31.6",
+    "ipfs": "github:ipfs/js-ipfs",
```

Comment:
This has been released, you can :shipit:  :D

Reply:
Yeah! Just found a problem blocking the interop to work 😞 

[js-ipfs#1570](https://github.com/ipfs/js-ipfs/pull/1570)

Reply:
@vasco-santos https://github.com/ipfs/js-ipfs/pull/1570 now merged and released, can this now be updated so we can merge?